Tests/MongoSwiftSyncTests/ClientSessionTests.swift

```diff
@@ -384,74 +400,76 @@ final class ClientSessionTests: MongoSwiftTestCase {
             }
             defer { center.removeObserver(replyObserver) }
 
-            _ = try collection.find(session: session).next()
+            _ = try collection.countDocuments(session: session)
```

Comment:
there are a lot of places in these tests where we need to do a read and were doing a find. I switched a lot of them to `countDocuments`for the purpose of enabling the tests as they don't seem to specify which read we do. do you remember if there was a particular reason you did `find`?

Reply:
I don't recall anything specific. If the spec doesn't say anything than `countDocuments` should be fine. Its just an aggregate under the hood anyways.